apigee.properties

```diff
@@ -2,11 +2,17 @@ category=External Analyzers
 description=Adds XML rules test Apigee apiproxies.
 homepageUrl=https://github.com/CreditMutuelArkea/sonar-apigee-plugin
 archivedVersions=1.1.0,1.2.0,1.3.0
-publicVersions=2.0.0
+publicVersions=2.1.0
 
 defaults.mavenGroupId=com.arkea.satd.sonar.apigee
 defaults.mavenArtifactId=sonar-apigee-plugin
 
+2.1.0.description=New features
```

Comment:
Really? Surely you want to give your users more than this?
Suggest: SharedFlow support, new rules.

Reply:
You're right. Done.